app/views/project/detail.scala.html

```diff
@@ -1,213 +1,195 @@
-@(project: models.Project, projectDetail: viewmodels.ProjectDetail, donateForm: Form[viewmodels.donation.CreateDonationData])
+@(widget: viewmodels.ProjectWidget, project: models.Project, donateForm: Form[viewmodels.donation.CreateDonationData], newsDataForm: Form[viewmodels.NewsData])
```

Comment:
Beim mergen wurde projectDetail viewmodel entfernt, dies wird aber in der Detailansicht benötigt